controls/stig_ctr.yml

```
@@ -0,0 +1,8 @@
+policy: Container Platform Security Requirements Guide
+title: Container Platform Security Requirements Guide
+id: stig_ocp4
```

```suggestion
id: srg_ctr
```

It looks like I made a small typo when giving you that command. The id should be `srg_ctr`. So the files and folder need to be moved, sorry about that.

---

No worries. On it.